aries-site/src/examples/components/datatable/DataTableMultiSelectExample.js

```diff
@@ -317,33 +317,33 @@ TableControls.propTypes = {
   selected: PropTypes.array,
 };
 
-const SelectionSummary = ({ selected }) => (
-  <>
-    {selected && (
-      <>
-        {selected.length > 0 ? (
-          <Box direction="row" gap="xxsmall">
-            <Text size="small" weight="bold">
-              {selected.length}
-            </Text>
-            <Text size="small">of</Text>
-            <Text size="small" weight="bold">
-              {data.length}
-            </Text>
-            <Text size="small">items selected</Text>
-          </Box>
-        ) : (
-          <Box direction="row" gap="xxsmall">
-            <Text size="small" weight="bold">
-              {data.length}
-            </Text>
-            <Text size="small">items</Text>
-          </Box>
-        )}
-      </>
-    )}
-  </>
-);
+const SelectionSummary = ({ selected }) => {
+  if (selected) {
```

Comment:
I don't think we want this change. Should disable this rule.

Reply:
Simplified the logic on this one.